e2e-tests/custom-login/specs/custom-login-flow-spec.js

```diff
@@ -106,9 +106,11 @@ describe('Custom Login Flow', () => {
 
     await authenticatorsPage.waitForPageLoad();
     authenticatorsPage.clickAuthenticatorByLabel('Email');
-
     await mfaChallengePage.waitForPageLoad();
 
+    // Wait for 5 seconds for email to be received
+    await browser.sleep(5000);
```

Comment:
Should this be configurable? What if it takes much longer?

Reply:
Added an env var to configure it.